gradle-plugins/biz.aQute.bnd.gradle/build.gradle.kts

```diff
@@ -77,7 +77,7 @@ dependencies {
 	implementation("biz.aQute.bnd:biz.aQute.resolve:${version}")
 	runtimeOnly("biz.aQute.bnd:biz.aQute.bnd.embedded-repo:${version}")
 	// keep in sync with cnf/junit.bnd e.g. 'junit.jupiter.version'
-	testImplementation(enforcedPlatform("org.junit:junit-bom:5.12.2"))
+	testImplementation(enforcedPlatform("org.junit:junit-bom:5.10.1"))
```

Comment:
If this change is necessary, then we need to change the comment: `eg 'junit.jupiter.tester.version'`

However, this again feels a bit wrong. Upgrading the Eclipse baseline shouldn't require us to downgrade JUnit.

Reply:
@kriegfrj just looked. Could it be that you reviewed an outdated commit? it is already at org.junit:junit-bom:5.12.2